apps/dialtone-documentation/docs/_data/site-nav.json

```
@@ -70,7 +96,7 @@
           "planned": true
         },
         {
-          "text": "Design assets",
+          "text": "Design Assets",
```

Don't think Assets need to be capitalized.

```suggestion
          "text": "Design assets",
```

---

Per the new writing guidance, menu labels should be title-cased

---

We need to change all the titles then, we were not capitalizing previously, doesn't need to be in this PR, but we should create a ticket to not forgot about it

---

We might as well scrub the current doc site to correct this. I'll do it. Also for action items such as buttons.